plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js

```diff
@@ -23,11 +23,6 @@ const containerSettings = {
     label: 'Grant Runtime Privileges',
     helpText: 'By default, containers are “unprivileged” and cannot, for example, run a Docker daemon inside a Docker container.',
     dockerOnly: 'Grant runtime privileges is only supported in Docker Runtime.'
-  },
-  forcePullImage: {
```

Comment:
Since we now only have `privileged` in the settings couldn't we just add that with its view declaration instead of doing it dynamically?